test/verify/check-machines

```diff
@@ -2710,10 +2710,6 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.click("#cpu-config-dialog-apply")
         b.wait_in_text("#vm-subVmTest1-cpu-model .pf-c-description-list__text", "custom (coreduo)")
 
-        # CPU configuration should be disabled for running VMs
-        b.click("#vm-subVmTest1-run")
-        b.wait_present("#vm-subVmTest1-cpu-model button.pf-m-disabled")
-
```

Comment:
Why is this change in commit `machine: Split xml editing function out of libvirt-common `?

Reply:
Right sorry:
moved that to: commit a5fc888c90c6bcfa0f60f59d59c904c18885d612
Author: Katerina Koukiou <kkoukiou@redhat.com>
Date:   Mon Nov 30 11:24:31 2020 +0100

    machines: Allow editing cpu mode/model even when the VM is running